proposals/0251-font-styles.md

```diff
@@ -28,16 +28,22 @@ The HMI should be able to provide a list of supported hyper text elements to SDL
 ```xml
 <struct name="TextField" since="1.0">
 :
-    <param name="hyperTextSupported" type="String" array="true" mandatory="false" minvalue="1" maxvalue="100" since="6.x">    
+    <param name="hyperTextSupported" type="String" array="true" mandatory="false" minsize="1" maxsize="100" since="6.x">    
```

Comment:
This needs a description tag explaining the contents like you laid out below. Something like "This parameter contains string elements of HTML elements which are designed for inline text, without any containing characters such as < or >. See this link for more details: https://developer.mozilla.org/en-US/docs/Web/HTML/Element#Inline_text_semantics"